.ci/release.sh

```diff
@@ -27,9 +27,14 @@ echo $PATH
 java -version
 
 set +x
-echo "--- Release the binaries to Maven Central :maven:"
-if [[ "${dry_run}" == "true" ]] ; then
-  ./mvnw -V -s .ci/settings.xml -Pgpg clean package --batch-mode | tee release.txt
-else
-  ./mvnw -V -s .ci/settings.xml -Pgpg clean deploy --batch-mode | tee release.txt
+# Default in dry-run mode
+GOAL="install"
```

Comment:
For dry-run, `package` is enough

```suggestion
GOAL="package"
```